protocol/communities/community.go

```diff
@@ -233,6 +259,14 @@ func (o *Community) CreateChat(chatID string, chat *protobuf.CommunityChat) (*Co
 		return nil, ErrChatAlreadyExists
 	}
 
+	var pos int32 = 0
+	for _, c := range o.config.CommunityDescription.Chats {
+		if c.CategoryId == chat.CategoryId {
```

Comment:
Probably I would add a comment explaining what is doing (setting pos == to numberOfChatsInCategory() + 1 ?)